src/platform/utilities/sso/mockKeepAliveSSO.js

```diff
@@ -0,0 +1,18 @@
+export default function keepAlive() {
```

Comment:
right now this is a mocked keepAlive call that is by default returning an active SSO session, hence some of the comments/flags above. I need to investigate a better way to toggle this for local testing

Reply:
I _think_ if the API is also running locally, the Flipper flag could be defaulted to off

Reply:
everything defaults to true for localhost feature toggles, but if you're logged in as any user, you can flip it.

this is covering a slightly different case. since our different SSO session logic is dependent on the keep-alive endpoint, and since that endpoint won't work locally because of CORS issues, there needs to be some mocking to ensure baseline functionality outside of auto login/logout